src/stuf/stuf_bg/tests/test_client.py

```diff
@@ -121,3 +121,6 @@ def test_getting_request_data_returns_valid_data(self):
                             f'Attributes "{subset}" produces an invalid StUF-BG. '
                             f"Error: {xmlschema.error_log.last_error.message}"
                         )
+                    for attribute in subset:
+                        with self.subTest(subset=subset, attribute=attribute):
+                            self.assertIn(attribute, data)
```

Comment:
This seemed like the best way to unit test against a bug like this happening.  The reason this test didn't fail before is the `voornamen` attribute would not be added but since it was still valid against the xsd the test would pass.

This will at least ensure the attribute is in the xml we will be using when making the request.